build.rs

```diff
@@ -391,6 +406,88 @@ fn get_version(src_dir: &Path) -> String {
     get_git_hash().expect("Could not get a version. Either set $FISH_BUILD_VERSION or install git.")
 }
 
+#[cfg(feature = "embed-data")]
+#[cfg(not(clippy))]
+/// Helper function for avoiding unnecessary rebuilds.
+/// Specify all files/directories which should trigger a rebuild if modified via `source_paths`.
+/// `cache_file` is used to store the time of the latest build.
```

Comment:
I wouldn't normally replicate variable names in comments.
But I suppose it is common Rust style, especially for doc comments, so maybe I'm on the wrong side of history.